### PR TITLE
Update cnc-mirrors.txt

### PR DESCRIPTION
**Pull request by kanet0x**

### CODE REVIEW

**Comment by pchote** [comment]
I have merged the other PRs here so that discussion can stay in the one place.

The `openra.koljasagorski.de` domain doesn't resolve for me, so these all appear to be dead links. Do these work for anyone else?

**Comment by jrb0001** [comment]
The name doesn't have an AAAA record and it doesn't work over IPv4 for me.

**Comment by obrakmann** [comment]
Confirming that HTTP over IPv4 times out against that host

**Comment by kanet0x** [comment]
Oh fuck, sorry guys. I forgot to change the domain record. Is fixed now! http://openra.koljasagorski.de

Best Regards from Germany, Kolja.

**Comment by jrb0001** [comment]
Works for me over IPv6 and IPv4.

File: content/packages/d2k-base-mirrors.txt [hunk 1]
```diff
@@ -2,3 +2,4 @@ http://openra.baxxster.no/openra/d2k-base.zip
 http://openra.ppmsite.com/d2k-base.zip
 http://openra.mirror.haffdata.com/d2k-base.zip
 http://dev.republic.community/h/openra/d2k-base.zip
+http://openra.koljasagorski.de/d2k-base.zip
```
Comment (obrakmann) [inline content/packages/d2k-base-mirrors.txt]: This file is missing

File: content/packages/d2k-patch106-mirrors.txt [hunk 2]
```diff
@@ -2,3 +2,4 @@ http://openra.baxxster.no/openra/d2k-patch106.zip
 http://openra.ppmsite.com/d2k-patch106.zip
 http://openra.mirror.haffdata.com/d2k-patch106.zip
 http://dev.republic.community/h/openra/d2k-patch106.zip
+http://openra.koljasagorski.de/d2k-patch106.zip
```
Comment (obrakmann) [inline content/packages/d2k-patch106-mirrors.txt]: This one as well

File: content/packages/d2k-quickinstall-mirrors.txt [hunk 3]
```diff
@@ -2,3 +2,4 @@ http://openra.baxxster.no/openra/d2k-quickinstall.zip
 http://openra.ppmsite.com/d2k-quickinstall.zip
 http://openra.mirror.haffdata.com/d2k-quickinstall.zip
 http://dev.republic.community/h/openra/d2k-quickinstall.zip
+http://openra.koljasagorski.de/d2k-quickinstall.zip
```
Comment (obrakmann) [inline content/packages/d2k-quickinstall-mirrors.txt]: and this

File: content/packages/fs-mirrors.txt [hunk 4]
```diff
@@ -2,3 +2,4 @@ http://openra.ppmsite.com/ts-expand.zip
 http://openra.mirror.haffdata.com/ts-expand.zip
 http://openra.baxxster.no/openra/ts-expand.zip
 http://dev.republic.community/h/openra/ts-expand.zip
+http://openra.koljasagorski.de/ts-expand.zip
```
Comment (obrakmann) [inline content/packages/fs-mirrors.txt]: and this

File: content/packages/fs-music-mirrors.txt [hunk 5]
```diff
@@ -2,3 +2,4 @@ http://openra.ppmsite.com/ts-fsmusic.zip
 http://openra.mirror.haffdata.com/ts-fsmusic.zip
 http://openra.baxxster.no/openra/ts-fsmusic.zip
 http://dev.republic.community/h/openra/ts-fsmusic.zip
+http://openra.koljasagorski.de/ts-fsmusic.zip
```
Comment (obrakmann) [inline content/packages/fs-music-mirrors.txt]: and this

File: content/packages/ra-cncdesert-mirrors.txt [hunk 6]
```diff
@@ -2,3 +2,4 @@ http://openra.baxxster.no/openra/ra-cncdesert.zip
 http://openra.ppmsite.com/ra-cncdesert.zip
 http://openra.mirror.haffdata.com/ra-cncdesert.zip
 http://dev.republic.community/h/openra/ra-cncdesert.zip
+http://openra.koljasagorski.de/ra-cncdesert.zip
```
Comment (obrakmann) [inline content/packages/ra-cncdesert-mirrors.txt]: and this

File: content/packages/ra-quickinstall-mirrors.txt [hunk 7]
```diff
@@ -2,3 +2,4 @@ http://openra.baxxster.no/openra/ra-quickinstall.zip
 http://openra.ppmsite.com/ra-quickinstall.zip
 http://openra.mirror.haffdata.com/ra-quickinstall.zip
 http://dev.republic.community/h/openra/ra-quickinstall.zip
+http://openra.koljasagorski.de/ra-quickinstall.zip
```
Comment (obrakmann) [inline content/packages/ra-quickinstall-mirrors.txt]: and this

File: content/packages/ra-scores-mirrors.txt [hunk 8]
```diff
@@ -2,3 +2,4 @@ http://openra.baxxster.no/openra/ra-scores.zip
 http://openra.ppmsite.com/ra-scores.zip
 http://openra.mirror.haffdata.com/ra-scores.zip
 http://dev.republic.community/h/openra/ra-scores.zip
+http://openra.koljasagorski.de/ra-scores.zip
```
Comment (obrakmann) [inline content/packages/ra-scores-mirrors.txt]: and this

File: content/packages/ts-quickinstall-mirrors.txt [hunk 9]
```diff
@@ -2,3 +2,4 @@ http://openra.ppmsite.com/ts-quickinstall.zip
 http://openra.mirror.haffdata.com/ts-quickinstall.zip
 http://openra.baxxster.no/openra/ts-quickinstall.zip
 http://dev.republic.community/h/openra/ts-quickinstall.zip
+http://openra.koljasagorski.de/ts-quickinstall.zip
```
Comment (obrakmann) [inline content/packages/ts-quickinstall-mirrors.txt]: and this

Comment (kanet0x) [reply content/packages/ts-quickinstall-mirrors.txt]: fixed

File: content/packages/ra-scores-mirrors.txt (hunk 8 above)
Comment (kanet0x) [reply content/packages/ra-scores-mirrors.txt]: fixed

File: content/packages/ra-quickinstall-mirrors.txt (hunk 7 above)
Comment (kanet0x) [reply content/packages/ra-quickinstall-mirrors.txt]: fixed

File: content/packages/ra-cncdesert-mirrors.txt (hunk 6 above)
Comment (kanet0x) [reply content/packages/ra-cncdesert-mirrors.txt]: fixed

File: content/packages/fs-music-mirrors.txt (hunk 5 above)
Comment (kanet0x) [reply content/packages/fs-music-mirrors.txt]: fixed

File: content/packages/fs-mirrors.txt (hunk 4 above)
Comment (kanet0x) [reply content/packages/fs-mirrors.txt]: fixed

File: content/packages/d2k-quickinstall-mirrors.txt (hunk 3 above)
Comment (kanet0x) [reply content/packages/d2k-quickinstall-mirrors.txt]: fixed

File: content/packages/d2k-patch106-mirrors.txt (hunk 2 above)
Comment (kanet0x) [reply content/packages/d2k-patch106-mirrors.txt]: fixed

File: content/packages/d2k-base-mirrors.txt (hunk 1 above)
Comment (kanet0x) [reply content/packages/d2k-base-mirrors.txt]: fixed

**Comment by abcdefg30** [comment]
What is the status here?

**Comment by pchote** [comment]
Looks like the links are 404, so closing due to lack of activity.

**Pull request closed without merging**